deploy/charts/openebs-monitoring/values.yaml

```diff
@@ -120,6 +144,12 @@ kube-prometheus-stack:
       - grafana-polystat-panel
       - snuids-trafficlights-panel
 
+alertRules:
+  pools:
+    enabled: true
+  volumes:
```

Comment:
Same comment as above for `volumes` also if it is specific to cstor volumes only.

Reply:
It is not specific to cStor volumes.

Reply:
if volumes and pools are using standard metrics - then we can keep them generic. For example when Mayastor or LVM rules are added, will the above still apply.

Reply:
If the answer to the above - is not clear or "no", then it makes sense to keep engine-specific rules under engine -- like cStorPool and cStorVolume.